yt_dlp/extractor/hearthisat.py

```diff
@@ -27,13 +23,12 @@ class HearThisAtIE(InfoExtractor):
             'title': 'Moofi - Dr. Kreep',
             'thumbnail': r're:^https?://.*\.jpg$',
             'timestamp': 1421564134,
-            'description': 'md5:a4b6190da9075c89b203ac369f24d66e',
+            'description': 'md5:1adb0667b01499f9d27e97ddfd53852a',
             'upload_date': '20150118',
-            'comment_count': int,
-            'view_count': int,
-            'like_count': int,
+            'playback_count': int,
+            'favoritings_count': int,
```

Comment:
see correct field names in common.py

Reply:
imo it's better to remove these and `categories` should be changed to `genre`.

View_count and playback count are diff
same with like_count and fav count

Reply:
they are not in the api request